src/Command.js

```diff
@@ -336,8 +336,22 @@ export default class Command {
 
       this[method]((err, completed, code = 0) => {
         if (err) {
-          log.error(method, "callback with error\n", err);
-          this._complete(err, 1, callback);
+          // If we have package details we can direct the developers attention
+          // to that specific package.
+          if (err.pkg) {
+            log.error(method, `Error occured with '${err.pkg.name}' while ` +
```

Comment:
We should be able to use `dedent` for this multi-line message. I'd also appreciate if this logic was extracted into a private instance method like `this._logPackageError(err);`